### PR TITLE
check for contacts dep

### PR DESCRIPTION
**Pull request by v1r0x**
see #59

### CODE REVIEW

File: templates/main.php [hunk 1]
```diff
@@ -79,6 +82,7 @@
 					<a id="tracking-settings">Location tracking settings</a>
 			</div>
 		</div>
+		<?php endif; ?>
```
Comment (jancborchardt) [inline templates/main.php]: This ending tag needs to be further up, otherwise it also hides the settings area.

**Comment by jancborchardt** [comment]
Other than the inline comment it looks ok. Please also review @DJaeger @brantje @Henni :)

**Comment by v1r0x** [comment]
Don't know how this happened :see_no_evil:
Thanks! Should be ok now :)

**Comment by jancborchardt** [comment]
Cool! Now it would be good if you can squash the 3 commits into one so we keep a clean history. :)

**Comment by v1r0x** [comment]
Is there an easy way or should I delete all 3 commits and commit a new one with all changes?

**Comment by Henni** [comment]
@v1r0x Search for 'rebase -i' or 'interactive rebase' in Google.

**Comment by v1r0x** [comment]
I hope this was right...

**Comment by DJaeger** [comment]
Looks good :+1:

**Pull request merged**